styles/tokens/font.css

```diff
@@ -0,0 +1,48 @@
+:root {
+  --diamond-font-family: -apple-system, system-ui, blinkmacsystemfont, roboto,
+    helvetica, arial, sans-serif;
+
+  --diamond-font-line-height: 1.5;
+  --diamond-font-line-height-sm: 1.25;
+
+  --diamond-font-size-default: 1rem;
+  --diamond-font-size-xs: 0.75em;
+  --diamond-font-size-sm: 0.875em;
+  --diamond-font-size-base: 0.875rem;
+  --diamond-font-size-md: 1.125em;
+  --diamond-font-size-lg: 1.25em;
+  --diamond-font-size-xl: 1.5em;
+  --diamond-font-size-xxl: 2em;
+  --diamond-font-size-xxxl: 3em;
+
+  --diamond-font-size-display1: 2rem;
```

Comment:
What are these display ones? They aren't used anywhere or linked to a normal element seems to be pre-empting a need that doesn't exist yet.

Reply:
I've had these bumped up h1/h2 for home or sales site areas on a few sites now. I'll remove them for the moment until we need them.